thrift/contentatom.thrift

```diff
@@ -0,0 +1,5 @@
+include "atoms/tenfour.thrift"
+
+union ContentAtom {
+  1: tenfour.Content tenfour
```

Comment:
What is the `tenfour` name here used for? I don't understand the semantics of Thrift `union` -- is it going to be the field name of a ContentAtom object, or is the object a `tenfour.Content` instance directly? Sorry I don't think I'm explaining myself well..